src/ImageSharp.Drawing/ImageSharp.Drawing.csproj

```diff
@@ -8,16 +8,40 @@
     <PackageId>SixLabors.ImageSharp.Drawing</PackageId>
     <PackageTags>Image Draw Shape Path Font</PackageTags>
     <RootNamespace>SixLabors.ImageSharp</RootNamespace>
-    <TargetFrameworks>netcoreapp2.1;netstandard1.3;netstandard2.0</TargetFrameworks>
-  </PropertyGroup>
 
-  <!-- TODO: Include .NETSTANDARD2.1 when released-->
-  <PropertyGroup Condition=" $(TargetFramework.StartsWith('netcoreapp2')) ">
-    <DefineConstants>$(DefineConstants);SUPPORTS_MATHF</DefineConstants>
+    <TargetFrameworks>netcoreapp2.1;netstandard2.0;netstandard1.3</TargetFrameworks>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" $(TargetFramework.StartsWith('netcoreapp2.1')) ">
-    <DefineConstants>$(DefineConstants);SUPPORTS_HASHCODE</DefineConstants>
+  <!--
```

Comment:
You can do that with csproj files?

Reply:
It could be moved to `.props` files.

First, I would place it to the one under the root of `src`, we can move contents to the shared one later, when split the repos.

Reply:
TIL

Reply:
TIL